index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Fulfillment App</title>
```

Comment:
We should follow storefront here, adding the metadata through meta data service rather than hardcode it here. 

Can you create a follow up task for that. It should also include a Spryker favicon.

Reply:
Created a ticket https://spryker.atlassian.net/browse/HRZ-90360